cmapPy/pandasGEXpress/__init__.py

```diff
@@ -1,4 +1 @@
-from .parse import parse
-#from .GCToo import GCToo
-#from .write_gctx import write_gctx
-#from .write_gct import write_gct 
+from cmapPy.pandasGEXpress.parse import parse
```

Comment:
@dllahr can you clarify why you changed this?

Reply:
For some reason the other way was causing tests to break, not sure why exactly

Reply:
Hm, weird/confused that that happened now but not previously

Reply:
I changed the import statements throughout the code to make them easier to update in the future - I really, really want to avoid "from ..." style imports.  Unfortunately it can't be avoided here.  I suspect that those changes might have caused the need to make the above change.

Reply:
Oh right, that could make sense!